src/Repository/AdminEntityRepository.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Repository;
+
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+
+abstract class AdminEntityRepository extends ServiceEntityRepository
+{
+    public function findList($limit, $offset)
+    {
+        return $this->createQueryBuilder('entity')
+            ->select('entity')
+            ->setFirstResult($offset)
+            ->setMaxResults($limit)
+            ->getQuery()
+            ->getResult();
+    }
+
+    public function countItems()
+    {
+        return $this->createQueryBuilder('entity')
+            ->select('count(entity.id)')
```

Comment:
@daria-sieroshtan 
Please use `$qb->expr()->count()`

Reply:
I prefer to keep this simple

Reply:
@daria-sieroshtan 
https://en.wikipedia.org/wiki/KISS_principle :)